lang/p4rs/src/table.rs

```diff
@@ -54,15 +77,10 @@ impl Key {
                     IpAddr::V6(a) => a.octets().into(),
                 };
                 v.push(p.len);
-                let w = v.len();
-                (v, w)
+                v.resize(v.len(), 0);
```

Comment:
I think this one is not necessary. It's essentially saying resize the vector to the size of the vector.

Reply:
Gotcha, this has been fixed.